traffic_monitor_golang/traffic_monitor/config/config.go

```diff
@@ -21,7 +21,6 @@ package config
 
```

Comment:
I disagree with TM changes being in the PR because one of the dependencies changed, the smaller we can keep this PR the easier it is to merge (and less conflicts) we have to deal with